### PR TITLE
Lint 2018 package

### PR DESCRIPTION
**Pull request by jaronheard**
Addresses #515 for 2018 package

### CODE REVIEW

**Comment by Dianna** [comment]
@jaronheard, doesn't look like I have the necessary permissions to assign myself as a reviewer. Let it be known that I shall, henceforth, review-ify these changes

File: packages/2018/src/components/HomePage.js [hunk 1]
```diff
@@ -697,10 +702,10 @@ class HomePage extends Component {
                   </p>
                 </div>
                 <div className={brainWrapper}>
-                  <img src={brain} />
+                  <img src={brain} alt="" />
```
Comment (Dianna) [inline packages/2018/src/components/HomePage.js]: Not sure what the page looks like, so depending on that an alt may be nice here

File: packages/2018/src/components/HomePage.js [hunk 2]
```diff
@@ -697,10 +702,10 @@ class HomePage extends Component {
...
                 </div>
                 <div className={brainWrapperMobile}>
-                  <img src={brainMobile} />
+                  <img src={brainMobile} alt="" />
```
Comment (Dianna) [inline packages/2018/src/components/HomePage.js]: Same about the alt

File: packages/2018/src/components/HomePage.js (hunk 1 above)
Comment (jaronheard) [reply packages/2018/src/components/HomePage.js]: Yeah, these are decorative, so empty alt is intentional! Thanks for thinking about it!

File: packages/2018/src/components/HomePage.js (hunk 2 above)
Comment (jaronheard) [reply packages/2018/src/components/HomePage.js]: Also decorative

**Comment by jaronheard** [comment]
Thanks for the review @Dianna! I've added you to the front-end team, so you have all the appropriate permissions now

**Pull request merged**